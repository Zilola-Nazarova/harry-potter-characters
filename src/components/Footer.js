import { Link } from 'react-router-dom';
import styles from '../styles/Footer.module.css';

const Footer = () => (
  <div className={styles.footer}>
    <p>Â© 2023 Harry Potter Library</p>
    <div className={styles.contacts}>
      <p>
        <a href="#/">Send a message</a>
        <Link to="/about">About</Link>
      </p>
    </div>
  </div>
);

export default Footer;
