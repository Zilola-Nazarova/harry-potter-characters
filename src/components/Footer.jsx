import { Link } from 'react-router-dom';
import styles from '../styles/Footer.module.css';

const Footer = () => (
  <div className={styles.footer}>
    <p>
      The weather forecast and all info about spots is provided for
      personal non-commercial use.
    </p>
    <p>
      Mountains Forecast does not promise any specific results from
      the use of its service.
    </p>
    <p>Â© 2023 Mountains Forecast</p>

    <div className={styles.contacts}>
      <p>
        <a href="#/">Send a message</a>
        <Link to="/about">About</Link>
      </p>
    </div>
  </div>
);

export default Footer;
