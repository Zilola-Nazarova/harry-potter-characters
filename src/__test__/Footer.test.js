import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

describe('Test Footer render', () => {
  test('Test Footer', () => {
    const component = renderer.create(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>,
    );
    expect(component).toMatchSnapshot();
  });
});
