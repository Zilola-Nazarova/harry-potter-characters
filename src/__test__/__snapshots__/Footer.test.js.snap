// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer render Test Footer 1`] = `
<div
  className="footer"
>
  <p>
    The weather forecast and all info about spots is provided for personal non-commercial use.
  </p>
  <p>
    Mountains Forecast does not promise any specific results from the use of its service.
  </p>
  <p>
    Â© 2023 Mountains Forecast
  </p>
  <div
    className="contacts"
  >
    <p>
      <a
        href="#/"
      >
        Send a message
      </a>
      <a
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
    </p>
  </div>
</div>
`;
